js/src/vm/StringType.h

```diff
@@ -1788,6 +1788,18 @@ template <AllowGC allowGC>
 extern JSString* ToStringSlow(
     JSContext* cx, typename MaybeRooted<Value, allowGC>::HandleType arg);
 
+/*
+ * Same as the 'ToStringSlow' function. The VMFUNCTION_LIST in
+ * VMFuncionList-inl.h cannot include any overloaded functions, so this name is
+ * provided for use in that list. ('ToStringSlow' has an overload in
+ * Conversions.h.)
```

Comment:
That's not a concern local to this file. I think it belongs in the VMFunctionList-inl.h to which it's pertinent?

Reply:
Agreed.